src/qgitref.h

```diff
@@ -141,11 +141,10 @@ namespace LibQGit2
              * to disk.
              *
              * @param target The new target for the reference
-             * @param signature The identity that will used to populate the reflog entry
              * @param message The one line long message to be appended to the reflog
              * @throws LibQGit2::Exception
              */
-            void setSymbolicTarget(const QString& target, const Signature &signature = Signature(), const QString &message = QString());
+            void setSymbolicTarget(const QString& target, const QString &message = QString());
```

Comment:
The `qgitsignature.h` include can be removed from this file.

Reply:
Removed.